tests/dialects/test_dialect.py

```diff
@@ -1763,13 +1763,52 @@ def test_operators(self):
             "LEVENSHTEIN(col1, col2)",
             write={
                 "bigquery": "EDIT_DISTANCE(col1, col2)",
```

Comment:
Let's add the same tests for the `read` dictionary as well, this way we can also test that for each dialect the parsing (and by extension, the transpilation) of Levenshtein works